contributors/cheatsheet.md

```diff
@@ -43,40 +43,53 @@ yo phovea:update
 Use a workspace
 --------------------
 
-A workspace helps if you are developing multiple plugins at the same time 
-and working with server side plugins using [Vagrant](https://www.vagrantup.com). 
+If you are developing multiple plugins at the same time a "workspace"
+using [Docker](https://www.docker.com/) and 
+[Docker Compose](https://www.docker.com/products/docker-compose)
+can make the process easier. Creating a workspace this way also generates a PyCharm project.
 The basic idea is to use the parent directory of the plugins, so that they can
-share a common npm installation and vagrant setup.
+share a common npm installation and docker setup.
+
+Make sure you have NPM, Yeoman, [Docker](https://docs.docker.com/engine/installation/)
+and [Docker Compose](https://docs.docker.com/compose/install/) installed.
 
 ```
 mkdir workspace
 cd workspace
-# initialize/clone/resolve plugins
+
+git clone https://github.com/phovea/phovea_server.git
+# AND/OR clone any other repos that you will need. 
+yo phovea:resolve
```

Comment:
@thinkh: Does it make sense to reference `phovea:resolve` at this point in the workflow?